app/views/comments/_mentionable_users.html.erb

```diff
@@ -0,0 +1,12 @@
+<%# this is picked up by javascripts/snowcrash/modules/comments.js.coffee %>
+<%= tag.div id: "mentionable-users",
```

Comment:
can this be cached? Otherwise we're making the same DB call to load all Users on every single page load.